microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/project/AbstractConfigSource.java

```diff
@@ -189,6 +189,11 @@ private void reset() {
 		return getPropertyInformations(propertyKey, getConfig());
 	}
 
+	@Override
+	public boolean isSameFile(IPath path) {

```

Comment:
It doesn't work on Windows OS since you compare IPathwith java.nio.Path. To fix that you should compare java.io.File. More getOutputConfigFile() can return null.

replace with 

```java
public boolean isSameFile(File file) {
	Path configFile = getOutputConfigFile();
	return configFile != null ? configFile.toFile().equals(file) : false;
}
```